src/utils/renderHelpers.ts

```diff
@@ -45,17 +45,20 @@ export function findFirstOccurrenceOutsideComment(
 }
 
 function findFirstLineBreakOutsideComment(code: string, start: number = 0) {
-	let commentStart, lineBreakPos;
+	let commentStart, lineBreakPos, nextChar;
```

Comment:
I take it the assumption in this function is that `start` is always at the end of all the code for a line?

Reply:
What happens in cases like:

```js
(code) /*

*/ (moar code)
```

Will this be able to deal with the start of a code block inline with the block comment?

Reply:
There is actually no assumption about `start`. Basically we
* find the next line-break following start
* find the next comment following start if it exists and see if it closes before we reach the line-break (if so, this is repeated to account for multiple comments)
* if the line-break is inside the comment, we find the end of the comment and repeat with the next line-break outside the comment

Thus in the situation above, the first line-break will be ignored and since there is not line-break between the end of the comment and the next statement, the algorithm will return -1.

Reply:
Ahh so the `code` passed to this function in my example above would only ever be the `" /*\n\n*/ "` part, that is it is guaranteed to be a substring between AST Nodes already?